templates/publisher/register-name.html

```diff
@@ -8,8 +8,13 @@
 <div class="p-strip">
 
   <div class="row">
-    <h1>Register a snap name</h1>
+    <h1 class="p-heading--three">Register a snap name</h1>
   </div>
+
+  <div class="row">
+    <h2 class="p-heading--four">Before you can push your snap to the store, its name must be registered</h2>
```

Comment:
`p-heading--four` seems almost the same as `--three`. Reducing it to `--five` looks better I think.